src/snowflake/snowpark/mock/mock_select_statement.py

```diff
@@ -70,9 +70,7 @@ def execution_plan(self):
 
     @property
     def attributes(self):
-        return self._attributes or (
-            self._execution_plan.attributes if self._execution_plan else None
-        )
+        return self._attributes or self.execution_plan.attributes
```

Comment:
`self._execution_plan` will always be non-None?

Reply:
Yes, `self.execution_plan` is a property that will now automatically initialize the field if it's `None`.